src/utils/platform.ts

```diff
@@ -70,3 +70,5 @@ export function isDefinedSelectorSupported() {
         return false;
     }
 }
+
+export const IS_SHADOW_DOM_SUPPORTED = typeof navigator !== 'undefined' && typeof ShadowRoot !== 'function';
```

Comment:
Navigator is not related to this constant. I was talking about other constants.

Reply:
Else Node.js fails :)